salt/modules/dockercompose.py

```diff
@@ -583,12 +606,16 @@ def restart(path, service_names=None):
     project = __load_project(path)
     debug_ret = {}
     result = {}
+    # TODO: needs work
     if isinstance(project, dict):
         return project
     else:
         try:
-            project.restart(service_names)
-            if debug:
+            if HAS_PYTHON_ON_WHALES:
+                project.compose.restart(services=service_names, quiet=True)
+            else:
+                project.restart(service_names)
+            if debug and not HAS_PYTHON_ON_WHALES:
```

Comment:
Can we add debug functionality for the python on whales library?

What are we adding todo's for this functionality? 

If we decide that it needs to be implemented later we need to atleast document it for each function that debug does not work when using python on whales.

Reply:
I have just completed this for every command